### PR TITLE
Prerequisites files code markdown format

### PR DESCRIPTION
**Pull request by tejash-wagento**
## This PR is a:
- [ ] New topic
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement
 

## Summary
Prerequisites files code markdown format 
When this pull request is merged, it will...

### CODE REVIEW

File: guides/v2.1/install-gde/prereq/apache.md [hunk 1]
```diff
@@ -111,10 +109,8 @@ To upgrade to Apache 2.4:
 
 		apt-get install -y apache2
 
-	<div class="bs-callout bs-callout-info" id="info">
-	<span class="glyphicon-class">
-	<p>If the <code>apt-get install</code> command fails because of unmet dependencies, consult a resource like <a href="http://askubuntu.com/questions/140246/how-do-i-resolve-unmet-dependencies-after-adding-a-ppa" target="_blank">http://askubuntu.com</a>.</p></span>
-	</div>
+	{:.bs-callout .bs-callout-info}
+	If the `apt-get install` command fails because of unmet dependencies, consult a resource like [http://askubuntu.com](http://askubuntu.com/questions/140246/how-do-i-resolve-unmet-dependencies-after-adding-a-ppa){:target="&#95;blank"}.
```
Comment (shrielenee) [inline guides/v2.1/install-gde/prereq/apache.md]: This looks strangely indented. Please verify.

File: guides/v2.1/install-gde/prereq/nginx.md [hunk 2]
```diff
@@ -16,8 +16,8 @@ Magento supports nginx 1.8 (or the [latest mainline version](http://nginx.org/en
 
 Installation instructions vary based on which operating system you're using:
```
Comment (shrielenee) [inline guides/v2.1/install-gde/prereq/nginx.md]: The bullet numbering in this section, under "For this example..." are improperly numbered. Please revise.

File: guides/v2.1/install-gde/prereq/php-centos.md [hunk 3]
```diff
@@ -42,12 +39,10 @@ To verify if PHP is installed already, enter `php -v`. If {% glossarytooltip bf7
 	Zend Engine v2.6.0, Copyright (c) 1998-2014 Zend Technologies
     with Zend OPcache v7.0.4-dev, Copyright (c) 1999-2014, by Zend Technologies
 
-<div class="bs-callout bs-callout-info" id="info">
-<span class="glyphicon-class">
-  <p>The preceding message confirms that the <code>Zend OPcache</code> is installed. We strongly recommend using the OPcache for performance reasons. If your PHP distribution does not come with the OPcache, see the <a href="http://php.net/manual/en/opcache.setup.php" target="&#95;blank">PHP OPcache documentation</a>.</p></span>
-</div>
+{:.bs-callout .bs-callout-info}
+The preceding message confirms that the <code>Zend OPcache</code> is installed. We strongly recommend using the OPcache for performance reasons. If your PHP distribution does not come with the OPcache, see the [PHP OPcache documentation](http://php.net/manual/en/opcache.setup.php){:target="&#95;blank"}.
```
Comment (shrielenee) [inline guides/v2.1/install-gde/prereq/php-centos.md]: "Zend OPcache" should be in backticks, not in `<code>`.

**Comment by shrielenee** [comment]
Just a few changes please @tejash-wagento - thanks!

**Comment by shrielenee** [comment]
Also please check that the `nginx.md` and `apache.md` files for v2.3 will receive this change if needed. It doesn't seem like they are symlinked to an earlier version.

**Comment by tejash-wagento** [comment]
Hi @shrielenee ,
I have completed changes as per your requirement.

**Comment by tejash-wagento** [comment]
Hi @shrielenee,
Changes is done.

File: guides/v2.1/install-gde/prereq/nginx.md (hunk 2 above)
Comment (tejash-wagento) [reply guides/v2.1/install-gde/prereq/nginx.md]: Above changes is done

**Pull request merged**